models/gpg_key.go

```diff
@@ -213,7 +213,7 @@ func parseGPGKey(ownerID int64, e *openpgp.Entity) (*GPGKey, error) {
 	for _, ident := range e.Identities {
 
 		for _, e := range userEmails {
-			if e.Email == ident.UserId.Email && e.IsActivated {
+			if e.Email == strings.ToLower(strings.TrimSpace(ident.UserId.Email)) && e.IsActivated {
```

Comment:
For little performance gain maybe cache strings.ToLower(strings.TrimSpace()) in a tmp variable and not do it at each userEmails.